fetch.bs

```diff
@@ -3811,6 +3814,44 @@ the request.
  <var>request</var>'s <a for=request>origin</a> to  <var>request</var>'s <a for=request>client</a>'s
  <a for="environment settings object">origin</a>.
 
+ <li><p>Let <var>hasCustomHeaders</var> be false.
+
+ <li><p><a for=list>For each</a> <var>header</var> in <var>request</var>'s
+ <a for=request>header list</a>, if <var>header</var>'s <a for=header>name</a> is
+ not `<code>If-Modified-Since</code>`, `<code>If-None-Match</code>`, `<code>If-Unmodified-Since</code>`,
+ `<code>If-Match</code>`, `<code>Origin</code>`, `<code>User-Agent</code>`, `<code>Referer</code>`,
+ or `<code>Cache-Control</code>`, set <var>hasCustomHeaders</var> to true.
```

Comment:
It cannot contain some of these headers at this point. E.g., Origin and Referer are forbidden.